File: Objective-C/Neocom/Neocom/NCImageFromDataValueTransformer.m
//
//  NCImageFromDataValueTransformer.m
//  Neocom
//
//  Created by Artem Shimanski on 23.11.16.
//  Copyright © 2016 Artem Shimanski. All rights reserved.
//

#import <UIKit/UIKit.h>
#import "NCImageFromDataValueTransformer.h"

@implementation NCImageFromDataValueTransformer

- (id) transformedValue:(id)value {
	return [UIImage imageWithData:value scale:UIScreen.mainScreen.scale];
}

@end

File: Neocom/Neocom/NCImageFromDataValueTransformer.swift
//
//  NCImageFromDataValueTransformer.swift
//  Neocom
//
//  Created by Artem Shimanski on 05.12.16.
//  Copyright © 2016 Artem Shimanski. All rights reserved.
//

import Foundation

class NCImageFromDataValueTransformer: ValueTransformer {
	override func transformedValue(_ value: Any?) -> Any? {
		guard let data = value as? Data else {return nil}
		return UIImage(data: data, scale: UIScreen.main.scale)
	}
}
